frontend/components/customizer/query-string-builder.js

```diff
@@ -175,10 +180,11 @@ export default class QueryStringBuilder extends React.Component {
         <td>
           {isStringParam ? (
             <QueryParamInput
-              checked={value}
               name={name}
               onChange={this.handleServiceQueryParamChange}
+              size={30}
               type="text"
+              value={value}
```

Comment:
This is a `text` input, so we should have a `value` instead of `checked` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/text).